### PR TITLE
fix: substringof filter was applied incorrectly

### PR DESCRIPTION
**Pull request by nlunets**
Instead of checking A exists withing B it was testing the otherway around leading to incorrect results.

### CODE REVIEW

File: packages/fe-mockserver-core/src/data/entitySets/entitySet.ts [hunk 1]
```diff
@@ -79,8 +79,11 @@ function transformationFn(type: string, check?: any) {
             return (data: string) => {
                 return data.endsWith(prepareLiteral(check, 'Edm.String') as string);
             };
-        case 'contains':
         case 'substringof':
+            return (data: string) => {
+                return check.indexOf(prepareLiteral(data, 'Edm.String') as string) !== -1;
```
Comment (erlethor) [inline packages/fe-mockserver-core/src/data/entitySets/entitySet.ts:84]: When is it `data` and when `check`?

Comment (nlunets) [reply packages/fe-mockserver-core/src/data/entitySets/entitySet.ts:84]: susbtringof(piece of string to check, string where to check) 

contains(string where to check, piece of string to check)

data equals the first arg
check equals the second arg

So for substringof check is the data model object and data is the static string to check (or whatever really)

Comment (erlethor) [reply packages/fe-mockserver-core/src/data/entitySets/entitySet.ts:84]: Okay. I didn't look at the `makeTransformationFn`. Now I get it (I hope 😄)

**Review by erlethor — APPROVED** [review]
LGTM

**Pull request merged**